consensus/poet/families/sawtooth_validator_registry/validator_reg_message_factory.py

```diff
@@ -287,3 +289,21 @@ def create_set_request_validator_map(self):
     def create_set_response_validator_map(self):
         addresses = [self._key_to_address("validator_map")]
         return self._factory.create_set_response(addresses)
+
```

Comment:
Okay.  I think I got the file mode fixed.  I had originally edited that file in Windows, which does't play nice with file mode.  I don't know if that is what caused it.